lib/sidekiq/batch/callback.rb

```diff
@@ -6,9 +6,10 @@ class Worker
 
         def perform(clazz, event, opts, bid)
           return unless %w(success complete).include?(event)
```

Comment:
It doesn't really care now about success or complete, so it won't work for
```ruby
batch.on(:complete, 'AnotherClass#method', 'uid' => current_user.id)
```
Will run both for `complete` and `success`